src/main/resources/templates/laDigitalAssister/householdUtilitiesAmount.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="fragments/head :: head(title=#{utilities-expenses-amount.title})"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="fragments/toolbar :: toolbar"></div>
+  <section class="slab">
+    <div class="grid">
+      <div th:replace="fragments/goBack :: goBackLink"></div>
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <p class="grey-text" th:text="#{utilities-expenses-amount.title}"></p>
+        <th:block
+            th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formDetails})}">
+          <th:block th:ref="formDetails">
+            <div class="form-card__content">
+              <div th:each="expense, iter:${inputData.get('householdUtilitiesExpenses[]')}"
+                   class="spacing-below-15">
+                <div th:if="${expense != 'Other'}">
```

Comment:
It seems like we should keep this in since there's a field for this in the PDF - was there a reason for leaving it out?

Reply:
@sree-cfa - the design says to ask the amount question for all expeneses except other: 
<img width="286" alt="Screenshot 2023-12-20 at 9 48 36 AM" src="https://github.com/codeforamerica/la-doc-uploader/assets/8609011/e856d475-7b03-40e3-bae4-850c4acc3f0d">

I can add it back in.